addons/event/models/event_tag.py

```diff
@@ -11,21 +11,26 @@ class EventTagCategory(models.Model):
     _description = "Event Tag Category"
     _order = "sequence"
 
+    def _default_sequence(self):
+        return (self.search([], order="sequence desc", limit=1).sequence or 0) + 1
+
     name = fields.Char("Name", required=True, translate=True)
-    sequence = fields.Integer('Sequence', default=0)
+    sequence = fields.Integer('Sequence', default=_default_sequence)
```

Comment:
Do we need this default trick?
Usually these types of issues are solved by giving a sort on the id after the sequence: 
`_order = "sequence, id"`